sdk/tables/aztable/connection_string_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We need tests for azurite connection strings also, since they do some different things with the account name.